vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/Interfaces.cs

```diff
@@ -49,7 +49,7 @@ bool BuildNestedProjectsOnBuild
     /// <summary>
     /// Interface for manipulating build dependency
     /// </summary>
-    /// <remarks>Normally this should be an /*internal, but public for FSharp.Project.dll*/ public interface but since it shouldbe available for the aggregator it must be made public.</remarks>
+    /// <remarks>Normally this should be an public interface but since it shouldbe available for the aggregator it must be made public.</remarks>
```

Comment:
The Api is a public Api, perhaps the comment probably needs an update and not just a removal of the commented out internal but in order to make sense.  If it were me, I would probably just remove the comment.